.github/workflows/build-lint-test.yml

```diff
@@ -20,8 +17,8 @@ jobs:
       - name: Install Yarn dependencies
         run: yarn --immutable
 
-  build:
-    name: Build
+  build-source:
```

Comment:
Ah, I forgot we recently merged in new changes to the workflows in the module template. They assume that the project has `build:source` and `build:types` scripts. There's more we would have to do to this project before we can completely carry these changes over, so I guess we'll have to revert some of the changes.

What if you return to the `build` job as you had before, but keep the changes to `node-version`? Sorry :(

Reply:
I've reverted some changes to bring back the original build job while keeping the updated node-version as you suggested.If there are any additional changes or further adjustments needed, please feel free to let me know